chain-signatures/node/src/indexer.rs

```diff
@@ -53,6 +54,14 @@ pub struct Options {
     /// The threshold in seconds to check if the indexer needs to be restarted due to it stalling.
     #[clap(long, env("MPC_INDEXER_RUNNING_THRESHOLD"), default_value = "300")]
     pub running_threshold: u64,
+
+    /// The threshold in block height lag to check if the indexer has caught up.
+    #[clap(
+        long,
+        env("MPC_INDEXER_BLOCK_HEIGHT_LAG_THRESHOLD"),
+        default_value = "50"
```

Comment:
hmm, not sure if 50 is the best. Might be too little to say it's behind. Have you tested longer lag threshold like 100 or 500?

Reply:
I was thinking about a lower number.
50 blocks is ~50 seconds. Are we often hitting that threshold?

If the node is 50 seconds behind, it fails all the assigned requests. Yes, it can still participate in other node protocols, but I would not consider it as "stable".

Reply:
I actually just looked at our dev where when it's generating signatures alright, the heights are within 50 from one another.

Reply:
I thought about it again. I think we should use a bigger value than 50. This lag is comparing with the latest block fetched from near rpc, and if lake has any delays, all nodes will be identified as unstable. 50s is probably too strict.
I'm thinking 200, as it is a bit smaller than the longest a signature request can wait in yield/resume, which means if nodes are <200 blocks behind, they'll still be able to answer the request in time.

Reply:
yup, lake can be delayed a fair bit, so yeah let's do something like 200. Lines up very well with yield/resume